linera-service/src/storage.rs

```diff
@@ -521,7 +521,9 @@ where
         StoreConfig::Memory(config, namespace) => {
             let store_config = MemoryStoreConfig::new(config.common_config.max_stream_queries);
             let mut storage = MemoryStorage::new(store_config, &namespace, wasm_runtime).await?;
-            genesis_config.initialize_storage(&mut storage).await?;
+            if let Some(genesis_config) = genesis_config {
```

Comment:
Or would it be easier to pass the genesis config to the proxy, too?
(It's even conceivable that in the future we'll add something to the genesis config that the proxy needs to know anyway?)